cmd/statusd/debug/debug.go

```diff
@@ -0,0 +1,107 @@
+package debug
+
+import (
+	"bufio"
+	"fmt"
+	"log"
+	"net"
+	"reflect"
+
+	"github.com/status-im/status-go/geth/api"
+)
+
+const (
+	// CLIPort is the CLI port.
+	CLIPort = "51515"
```

Comment:
I guess it's used only in `debug_test.go` now so it can be moved there.

Reply:
It's the default port.